src/module.ts

```diff
@@ -124,7 +124,10 @@ export default defineNuxtModule({
         _viewerDevMiddleware(req, res)
       }
       addServerMiddleware({ path, handler: viewerDevMiddleware })
-      nuxt.hook('listen', () => { logger.info(`Tailwind Viewer: ${chalk.underline.yellow(path)}`) })
+      nuxt.hook('listen', () => {
+        const url = withTrailingSlash(joinURL(nuxt.server.listeners && nuxt.server.listeners[0] ? nuxt.server.listeners[0].url : "/", path))
+        nuxt.options.cli.badgeMessages.push(`${chalk.bold('Tailwind Viewer: ')}${chalk.underline.yellow(url)}`)
```

Comment:
This major version is written for Nuxt 3 with nuxi CLI which does not provide banner feature anymore

Reply:
add `isNuxt2` will be fine. <br />but is there any plan to restore to old style?

Reply:
i searched the doc and found that with the the key `nuxt.options.bridge`, we'd differ the pure nuxt2 from others. <br />
```
      nuxt.hook('listen', (_, listener) => {
        const url = withTrailingSlash(joinURL(listener.url, path))
        if (isNuxt2() && !(nuxt.options as any).bridge) {
          nuxt.options.cli.badgeMessages.push(`Tailwind Viewer: ${chalk.underline.yellow(url)}`)
        } else {
          logger.info(`Tailwind Viewer: ${chalk.underline.yellow(url)}`)
        }
      })
```
<br />
tested passed under nuxt2, nuxt2+bridge, nuxt3.<br />
it looks more nice, but a little redundant